platform/darwin/src/MGLMultiPoint.mm

```diff
@@ -128,4 +128,10 @@ - (void)addShapeAnnotationObjectToCollection:(std::vector<mbgl::ShapeAnnotation>
     return mbgl::ShapeAnnotation::Properties();
 }
 
+- (NSString *)description
+{
+    return [NSString stringWithFormat:@"<%@: %p; count = %zu; bounds.ne = %f, %f; bounds.sw = %f, %f>",
```

Comment:
`%zu` is technically correct, since mbgl uses a `size_t`-typed count, but the `_count` ivar is `NSUInteger`-typed, so this format should use `%lu` with a cast to `(unsigned long)`.